extensions/sql-migration/src/dialog/assessmentResults/sqlDatabasesTree.ts

```diff
@@ -215,6 +219,25 @@ export class SqlDatabaseTree {
 		return searchContainer;
 	}
 
+	@debounce(500)
+	private _filterTableList(value: string): void {
+		if (this._databaseTableValues && value && value.length > 0) {
```

Comment:
FYI you can use null-coalescing to shorten checks like this `if (this._databaseTableValues && value?.length > 0)`